tests/test_hdf5.py

```diff
@@ -6,11 +6,11 @@
 from numpy.testing import assert_equal, assert_raises
 from six.moves import range, cPickle
 
-from fuel.datasets.hdf5 import Hdf5Dataset, H5PYDataset
+from fuel.datasets.hdf5 import PytablesDataset, H5PYDataset
 from fuel.streams import DataStream
 
 
-class TestHdf5Dataset(object):
+class TestPytablesDataset(object):
```

Comment:
nosetests also recognizes test classes and test class methods by name (see [here](http://nose.readthedocs.org/en/latest/writing_tests.html#test-classes)).